spec/samples/configuration/.reek

```diff
@@ -0,0 +1,9 @@
+---
+IrresponsibleModule:
+  enabled: false
+"spec/samples/three_clean_files/":
+  UtilityFunction:
+    enabled: false
+exclude_paths:
+  - "spec/samples/two_smelly_files/"
+  - "spec/samples/source_with_non_ruby_files/"
```

Comment:
This file is confusing since it seems like a grabbag for all kind of things. Why do we need it? By whom is it used for what?